draft-linker-adem-wg-adem-core.md

```diff
@@ -387,6 +409,8 @@ The order of these values encodes the *strength* of the verification result.
 7. `ENDORSED-UNTRUSTED`
 8. `ENDORSED-TRUSTED`
 
+TODO(Djackson) - Doesn't the verification algorithm also need a list of all acceptable pulic keys? 
```

Comment:
Is this comment on *list* of all acceptable public keys? I think, this is considerably easier to specify for one public key. People can make the checks for lists if they want.